ntbs-service/Pages/Shared/_Footer.cshtml

```diff
@@ -24,7 +24,7 @@
                             </a>
                         </li>
                         <li class="govuk-footer__inline-list-item">
-                            <a class="govuk-footer__link footer-link" href="#3">
+                            <a class="govuk-footer__link footer-link" href="https://phecloud.sharepoint.com/sites/NTBSResources/SitePages/NTBS-Accessibility-Statement.aspx" target="_blank" rel="noopener noreferrer">

```

Comment:
Do you think it would be worth making this configurable?

Reply:
Yeah it would be. I've generalised all footer URLs to be got from `appsettings.json` as we discussed